### ISSUE TITLE
Automatic links to existing pages?

### ISSUE THREAD

**Issue by zefhemel** [silverbulletmd/silverbullet#480]
### Discussed in https://github.com/silverbulletmd/silverbullet/discussions/460

<div type='discussions-op-text'>

<sup>Originally posted by **TheNomad11** July  9, 2023</sup>
There is alrealdy a kind of automatic linking to existing pages if you start with `[[` - it would list existing pages, but it would be even more convenient if one would just write a text, save it and then voila Silverbullet would automatically insert links to existing page titles that occur in my text as in Mediawiki https://www.mediawiki.org/wiki/Extension:LinkTitles I find this extension extremely useful for connecting existing content. What do you think? </div>

**Comment by zefhemel** [comment]
Many PKM systems have a feature called “unlinked mentions” which is exactly what you mean here. I think this can be done, either inline or in a separate panel (like most other PKMs like Obsidian and LogSeq) do. Some rules would need to apply, for instance if you happen to create a page named `a` you probably don’t want every instance of the word “a” in your space to link to that page, but I’m sure there’s some filtering that can be done here (such as a minimum length).

**Comment by johnlunney** [comment]
Would "unlinked mentions" make sense as part of the existing Backlinks plug? Excited for this!

**Comment by zefhemel** [comment]
Yes I'm thinking of adding an additional panel at the bottom of the page that could both show backlinks and not linked mentions as well.
